src/mobile/ios/iotaWallet.xcodeproj/project.pbxproj

```diff
@@ -1798,6 +1819,10 @@
 					ProductGroup = 601BFD5D20C39A560097D329 /* Products */;
 					ProjectRef = 601BFD5C20C39A560097D329 /* RNViewShot.xcodeproj */;
 				},
+				{
+					ProductGroup = 80B47902221B5ADF00D8BBE0 /* Products */;
+					ProjectRef = 3642AD7F4F9145FE803D1120 /* Signing.xcodeproj */;
```

Comment:
The signing package might need to be relinked on iOS because of https://github.com/iotaledger/iota.js/pull/355/commits/a810b0ba51d9dbd8746acd31a1b95616bd32cb0a